app/presenters/hyku/manifest_enabled_work_show_presenter.rb

```diff
@@ -35,8 +35,32 @@ def manifest_metadata
       metadata
     end
 
+    # assumes there can only be one doi
+    def doi
+      doi_regex = %r{10.\d{4,9}\/[-._;()\/:A-Z0-9]+}i
+      doi = extract_from_identifier(doi_regex)
+      doi.join unless doi.nil?
+    end
+
+    # unlike doi, there can be multiple isbns
+    def isbns
+      isbn_regex = /(?:ISBN[- ]*13|ISBN[- ]*10|)\s*
+                   ((?:(?:9[\s-]*7[\s-]*[89])?[ -]?(?:[0-9][ -]*){9})[ -]*(?:[0-9xX]))/x
+      isbns = extract_from_identifier(isbn_regex)
+      isbns.flatten unless doi.nil?
```

Comment:
looks like this should be `unless isbns.nil?` or `if isbns`.  Looks like doi is sneaking in here.

Reply:
Ooops! Thank you.